src/components/BackButton/StyledBackButton.tsx

```diff
@@ -0,0 +1,20 @@
+import { Button } from '..';
+import styled from 'styled-components';
+import BackLight from '../../views/HowToPlay/assets/BackLight.svg';
+import colors from '@ui/styles/colors';
+
+export const StyledBackButton = styled(Button)`
+  border-radius: 5px;
```

Comment:
Let's round this out just a bit more

```suggestion
  border-radius: 10px;
```